content/doc/book/getting-started/installing.adoc

```diff
@@ -101,6 +101,58 @@ If your `/etc/init.d/jenkins` file fails to start Jenkins, edit the `/etc/defaul
 Here, "8081" was chosen but you can put another port available.
 ====
 
+==== OpenIndiana/Hipster
+On a system running the OpenIndiana/Hipster you can install Jenkins in either local or global zone using the IPS packaging system.
+
+For the common case of running the newest packaged weekly build as a standalone (Jetty) server, simply execute:
+[source,bash]
+----
+pkg install jenkins
+svcadm enable jenkins
+----
+
+The common packaging integration for standalone service will:
+
+* Create a `jenkins` user to run the service and to own the directory structures under `/var/lib/jenkins`.
+* Pull the OpenJDK8 and other packages required to execute Jenkins, including the `jenkins-core-weekly` package with the latest `jenkins.war` (note: Jenkins LTS releases currently do not support OpenZFS-based systems well, so no packaging is provided at this time).
+* Setup Jenkins as an `svc:/network/http:jenkins` SMF service; you need to `enable` it as shown above, if you want the server to be launched on every start of the zone. See `/lib/svc/manifest/network/jenkins-standalone.xml` for more details.
+* The console log output is managed by SMF at `/var/svc/log/network-http:jenkins.log`. Check this file if you are troubleshooting Jenkins, as well as to find the initial-setup generated administrator password.
+* Set Jenkins to listen on port 8080. Access this port with your browser to start configuration.
+* If you want to change some service attributes, such as the environment variables (e.g. a custom `JENKINS_HOME`), the listen port or other aspects of the command-line invokation, you can run:
+[source,bash]
+----
+svccfg -s svc:/network/http:jenkins editprop
+svcadm refresh svc:/network/http:jenkins
+----
+
+Other notes:
+
+* Subsequent upgrades should be as simple as `pkg update` (for the whole operating environment), or to specifically update the Jenkins core:
+[source,bash]
+----
+pkg update jenkins-core-weekly
+----
+
+* Note that the package updating procedure would restart your packaged instance of the Jenkins application server.
+
+* To review other packages available for Jenkins integration in your release of OpenIndiana, run:
+[source,bash]
+----
+pkg info -r '*jenkins*'
+----
+
+==== Solaris, OmniOS, SmartOS and other siblings
```

Comment:
Maybe it makes sense to add a "This platform is NOT officially supported, use it at your own risk" disclaimer.